co_sim_io/co_sim_io.hpp

```diff
@@ -43,59 +43,53 @@ namespace CoSimIO {
 inline ReturnInfo Hello();
 
 
-inline ReturnInfo Connect(const std::string& rConnectionName, CoSimIO::SettingsType Settings);
-inline ReturnInfo Connect(const std::string& rConnectionName, const std::string& rSettingsFileName);
+inline ReturnInfo Connect(
+    const ConnectionSettings& I_Settings);
 
-inline ReturnInfo Disconnect(const std::string& rConnectionName);
+inline ReturnInfo Disconnect(
```

Comment:
Shall we pass here the `ConnectionSettings` instead of Info?

Reply:
hmm good question
I am not sure
For `Connect` we use `ConnectionSettings` because we might want to pass "extra" things like an `MPI_Comm`
this is not needed for `Disconnect`

So I am not sure

Reply:
So let's leave it as it is